### ISSUE TITLE
Import tensorflow failed

### ISSUE THREAD

**Issue by hrishi16** [BhallaLab/DeepCutAnalysis#2]
"
root@gulgula:~/Work/DeepCutAnalysis# python work_script.py 
Failed to import tensorflow
Traceback (most recent call last):
  File "work_script.py", line 11, in <module>
    raise e
ImportError: No module named tensorflow
"

**Comment by hrishi16** [comment]
Tried these:

root@gulgula:~/Work# pip install deeplabcut
Requirement already satisfied: deeplabcut in /usr/local/lib/python3.6/dist-packages (2.0.5)

and 

root@gulgula:~# pip install tensorflow-gpu
Requirement already satisfied: tensorflow-gpu in /usr/local/lib/python3.6/dist-packages (1.8.0)

and 

root@gulgula:~# pip install tensorflow-gpu --ignore-installed
Collecting tensorflow-gpu
  Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f3bb9bfd630>: Failed to establish a new connection: [Errno 101] Network is unreachable',)': /simple/tensorflow-gpu/

the connection keeps failing

**Comment by dilawar** [comment]
tensorflow will work only inside the docker image. Not in the host system. It has not been configured and installed to work with GPU yet. The whole point of docker image is that we don't have to build and install anything locally.

Run it inside the docker image, if it fails then we are in trouble.

**Comment by dilawar** [comment]
I see. The problem is with docker image.

**Comment by dilawar** [comment]
I have update the Dockerfile and added a script to install wxpython.

```
apt install \
    dpkg-dev \
    build-essential \
    libjpeg-dev \
    libtiff-dev \
    libsdl1.2-dev \
    libgstreamer-plugins-base0.10-dev \
    libnotify-dev \
    freeglut3 \
    freeglut3-dev \
    libsm-dev \
    libgtk-3-dev \
    libwebkitgtk-3.0-dev \
    libxtst-dev \
    python3-tk
python3 -m pip install wxpython
```

Just rebuild the dockerfile and run it again. It should fix it.

**Comment by hrishi16** [comment]
after git pull and running the build_image.sh - this is the output
 

/bin/sh: 1: ./install_wxpython.sh: not found
The command '/bin/sh -c ./install_wxpython.sh' returned a non-zero code: 127

**Comment by dilawar** [comment]
Recent  commit f848e3306a9509570d2dbc6bcc040d58801705aa fixes #2.
